test/unit/lib/database.ttest.js

```diff
@@ -63,6 +63,8 @@ test('Mongo Database tests', async dbTest => {
     }
   })
 
+  const timeout = ms => new Promise((resolve, reject) => setTimeout(resolve, ms))
```

Comment:
Does this ever get used in this file?

Also, was there a reason for changing the filename to `database.ttest.js`?